src/Groups/matrices/iso_nf_fq.jl

```diff
@@ -0,0 +1,153 @@
+# Detinko, Flannery, O'Brien "Recognizing finite matrix groups over infinite
+# fields", Section 4.2
+function isomorphic_group_over_finite_field(matrices::Vector{T}) where T <: Union{ fmpq_mat, AbstractAlgebra.MatElem{nf_elem} }
+   @assert !isempty(matrices)
```

Comment:
None of this code is tested right now. Could you please add some tests? Even if it's just some simple 2x2 integer matrices (could try both a finite and an infinite one).

Reply:
I wanted to wait until the interface is a little bit more polished, but sure. Never too early to test. Now the tests always print this "#I  Forcing finiteness test" when I call GAP.Globals.IsomorphismGroups (I can't call isisomorphic because one of them only exists on the GAP side). Is there a way to suppress this?

Reply:
I replaced it by `GroupHomomorphismByImages` (as suggested by you).